db/schema.rb

```diff
@@ -504,7 +504,7 @@
 
   create_table "feature_flags", force: true do |t|
     t.string  "key"
-    t.boolean "enabled"
+    t.boolean "enabled", default: false
```

Comment:
I vaguely remember us committing this separately?  Could you rebase onto master to check?